samples/java-spring-cinema-booking-choreography/src/main/java/com/example/cinema/model/CinemaApiModel.java

```diff
@@ -0,0 +1,65 @@
+package com.example.cinema.model;
+
+import com.fasterxml.jackson.annotation.JsonSubTypes;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+import java.util.List;
+
+public interface CinemaApiModel {
+    sealed interface ShowCommand {
```

Comment:
Maybe not needed to have a ShowCommand. It's not adding any extra value.

Reply:
But it is maybe convenient for code organization

Reply:
But since the idea is to move commands and events inside the model (as inner records), we don't need to wrap them once more.